server/paths.ts

```diff
@@ -124,13 +124,15 @@ export const getStaticPathsForDocs = (): DocsStaticPath[] => {
 
   versions.forEach((version) => {
     result.push(
-      ...getSlugsForVersion(version).map((path) => {
-        const slug = path.split("/").filter((part) => part);
-
-        return {
-          params: slug.length ? { slug } : { slug: undefined },
-        };
-      })
+      ...getSlugsForVersion(version)
+        .filter((path) => path !== "/older-versions/") // We should remove older-versions.mdx from docs
```

Comment:
We can remove this check once https://github.com/gravitational/teleport/pull/23192 gets pulled in. But good to know we have a killswitch for anything similar in the future. thank you!